styles/Vocab/renovate/accept.txt

```diff
@@ -0,0 +1,67 @@
+# This is the list of acceptable vocab for the Renovate project.
+
+# By default all entries are case sensitive!
+# Use (?i) to mark the entire pattern case insensitive.
+# Use [Oo]bservability to accept both observability and Observability.
+
+# Docs: https://docs.errata.ai/vale/vocab
+
+[Aa]utodetect
+[Aa]utomerge
+[Aa]utomerging
+[Cc]odeowners
+[Cc]onfig
+[Dd]atasource
+[Ff]unction
+[yaml, YAML]
+AdBlock
+Ansible
+Artifactory
+breejs
+Bundler
+cron
+Dependabot
+Docker
+Dockerfile
+dotnet
+ESLint
+Gemfile
+Gitea
+GitHub
```

Comment:
The words in this list are case sensitive by default, so here I'm enforcing that we always spell `GitHub` in the proper way. 😄 

Vale ignores GitHub Flavoured Style codeblocks, so it won't complain if we use `github` in a Renovate config example.